### PR TITLE
[FIX] *: update document links

### PR DESCRIPTION
**Pull request by sswapnesh**
Before this commit, Documents were
referenced for the 13.0 version instead of the current one. In this
commit, the references are corrected.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by odony** [comment]
Thanks, but actually we have an automatic redirect from /documentation/user/* into the last stable version of the documentation. For example https://www.odoo.com/documentation/user/foo/bar.html will automatically redirect to https://www.odoo.com/documentation/user/14.0/foo/bar.html now that 14.0 is released.

So I suggest we remove all the hardcoded "13.0" references instead of replacing them with "14.0".

It may be a little bit less accurate to always link to the last version, but I think it's ok because the version switcher is always visible, so even when the latest becomes 16.0 our users will be able to locate the 14.0 doc easily.

If we do that we will save the trouble of maintaining these links at every release ;-)

Thanks!

**Comment by sswapnesh** [comment]
@odony  Updated.

**Comment by AntoineVDV** [comment]
> So I suggest we remove all the hardcoded "13.0" references instead of replacing them with "14.0".

@odony @sswapnesh The hardcoded version was intended in the original spec (task 2180574) so that an Odoo 13 user would always land on the version 13.0 of the documentation rather than on the latest deployed version.
As the documentation content can vary widely from one version to another, I suggest that we don't remove the version string from the URLs but simply bump it to "14.0".

**Comment by sswapnesh** [comment]
I agree with @AntoineVDV We should use reference of current version as it might be confusion for users.

**Comment by odony** [comment]
> @odony @sswapnesh The hardcoded version was intended in the original spec (task 2180574) so that an Odoo 13 user would always land on the version 13.0 of the documentation rather than on the latest deployed version.

With the result that users of 14.0 end up with the 13.0 documentation, which is clearly better :p

**Comment by sswapnesh** [comment]
@odony @AntoineVDV  What was the Final decision ?
(I would go for making them 14.0 so in future users of >14 do not land on Latest version's documentation and open Issue here to fix it 😉 )

**Comment by AntoineVDV** [comment]
> (I would go for making them 14.0 so in future users of >14 do not land on Latest version's documentation and open Issue here to fix it  )

Fine by me :+1: 
We should just make sure next year to update those links before the release of Odoo 15.
The current situation of users being redirected to the doc of Odoo 13 is not a big deal since it shares 99.9% of its content with the 13.0. Having Odoo 14 users being redirected to the doc of Odoo 16 in 2 years would be much more confusing.

**Comment by sswapnesh** [comment]
I will wait for answer from @odony  Before making new commit.

**Comment by odony** [comment]
> I will wait for answer from @odony Before making new commit.

Go ahead, your rationale makes sense ;-)

**Comment by sswapnesh** [comment]
Should be good for r+ now

File: addons/website/views/snippets/snippets.xml [hunk 1]
```diff
@@ -1071,7 +1071,7 @@
     <div data-js="ThemeColors" data-selector="theme-colors" data-no-check="true">
         <we-alert class="o_old_color_system_warning d-none">
             It appears your website is still using the old color system of
-            Odoo 13.0 in some places. We made sure it is still working but
+            Odoo 14.0 in some places. We made sure it is still working but
```
Comment (AntoineVDV) [inline addons/website/views/snippets/snippets.xml:1074]: Given the surrounding text, it appears that it should stay "13.0" in this case.

File: doc/setup/email_gateway.rst [hunk 2]
```diff
@@ -40,4 +40,4 @@ For Exim
     - `Exim <https://www.exim.org/docs.html>`
 
 
-.. note:: If you don't have access/manage your email server, use `inbound messages<https://www.odoo.com/documentation/user/13.0/discuss/email_servers.html#how-to-manage-inbound-messages>`.
+.. note:: If you don't have access/manage your email server, use `inbound messages<https://www.odoo.com/documentation/user/14.0/discuss/email_servers.html#how-to-manage-inbound-messages>`.
```
Comment (AntoineVDV) [inline doc/setup/email_gateway.rst:43]: Self-reminder to fix this in 12 @AntoineVDV

**Review by AntoineVDV — COMMENTED** [review]
Thank you for your contribution! :+1: 
Please just check the comment below.

File: addons/website/views/snippets/snippets.xml (hunk 1 above)
Comment (sswapnesh) [reply addons/website/views/snippets/snippets.xml]: @AntoineVDV  Updated

**Comment by mart-e** [comment]
Thanks for the update :+1: 
@robodoo r+

**Pull request closed without merging**